index.js

```diff
@@ -141,44 +193,65 @@ const line = []
 
 
 // *** Uncomment the lines below to test
-// console.log("%cScope & Closures - Question 1", "color: red")
+console.log("%cScope & Closures - Question 1", "color: red")
+
+let currentLine = 0;
 
-// console.log(takeATicketNumber(line))
+function takeATicketNumber(line) {
+  currentLine++
+  line.push(currentLine)
+  return `Welcome. You are ticket number ${currentLine}`
+}
+
```

Comment:
good use of global scope here to make sure your variable does not reset with each function call.